@pferraro I'm now getting the following in server.log:

```
Caused by: jakarta.persistence.PersistenceException: [PersistenceUnit: CTS-EM2] Unable to build Hibernate SessionFactory
	at org.hibernate@5.3.24.Final//org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.persistenceException(EntityManagerFactoryBuilderImpl.java:1327)
	at org.hibernate@5.3.24.Final//org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.build(EntityManagerFactoryBuilderImpl.java:1253)
	at org.hibernate.jipijapa-hibernate5-3@27.0.0.Beta1-SNAPSHOT//org.jboss.as.jpa.hibernate5.TwoPhaseBootstrapImpl.build(TwoPhaseBootstrapImpl.java:44)
	at org.jboss.as.jpa@27.0.0.Beta1-SNAPSHOT//org.jboss.as.jpa.service.PersistenceUnitServiceImpl$1$1.run(PersistenceUnitServiceImpl.java:171)
	... 10 more
Caused by: org.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.cache.spi.CacheImplementor]
	at org.hibernate@5.3.24.Final//org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:275)
	at org.hibernate@5.3.24.Final//org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:237)
	at org.hibernate@5.3.24.Final//org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:214)
	at org.hibernate@5.3.24.Final//org.hibernate.service.internal.SessionFactoryServiceRegistryImpl.getService(SessionFactoryServiceRegistryImpl.java:109)
	at org.hibernate@5.3.24.Final//org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:239)
	at org.hibernate@5.3.24.Final//org.hibernate.boot.internal.SessionFactoryBuilderImpl.build(SessionFactoryBuilderImpl.java:467)
	at org.hibernate@5.3.24.Final//org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.build(EntityManagerFactoryBuilderImpl.java:1250)
	... 12 more
Caused by: org.hibernate.cache.CacheException: HHH025009: Unable to start region factory
	at org.infinispan.hibernate-cache@13.0.6.Final//org.infinispan.hibernate.cache.v53.InfinispanRegionFactory.start(InfinispanRegionFactory.java:248)
	at org.hibernate@5.3.24.Final//org.hibernate.cache.internal.EnabledCaching.<init>(EnabledCaching.java:82)
	at org.hibernate@5.3.24.Final//org.hibernate.engine.spi.CacheInitiator.initiateService(CacheInitiator.java:33)
	at org.hibernate@5.3.24.Final//org.hibernate.engine.spi.CacheInitiator.initiateService(CacheInitiator.java:24)
	at org.hibernate@5.3.24.Final//org.hibernate.service.spi.SessionFactoryServiceInitiator.initiateService(SessionFactoryServiceInitiator.java:30)
	at org.hibernate@5.3.24.Final//org.hibernate.service.internal.SessionFactoryServiceRegistryImpl.initiateService(SessionFactoryServiceRegistryImpl.java:68)
	at org.hibernate@5.3.24.Final//org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:263)
	... 18 more
Caused by: org.infinispan.commons.CacheConfigurationException: ISPN000282: Cannot enable Invocation Batching when the Transaction Mode is NON_TRANSACTIONAL, set the transaction mode to TRANSACTIONAL
	at org.infinispan@13.0.6.Final//org.infinispan.configuration.cache.InvocationBatchingConfigurationBuilder.validate(InvocationBatchingConfigurationBuilder.java:57)
	at org.infinispan@13.0.6.Final//org.infinispan.configuration.cache.ConfigurationBuilder.validate(ConfigurationBuilder.java:180)
	at org.infinispan@13.0.6.Final//org.infinispan.configuration.cache.ConfigurationBuilder.build(ConfigurationBuilder.java:236)
	at org.infinispan@13.0.6.Final//org.infinispan.configuration.cache.ConfigurationBuilder.build(ConfigurationBuilder.java:226)
	at org.infinispan.hibernate-cache@13.0.6.Final//org.infinispan.hibernate.cache.v53.InfinispanRegionFactory.defineDataTypeCacheConfigurations(InfinispanRegionFactory.java:374)
	at org.infinispan.hibernate-cache@13.0.6.Final//org.infinispan.hibernate.cache.v53.InfinispanRegionFactory.start(InfinispanRegionFactory.java:242)
	... 24 more

```

In case it helps, my standalone.xml has:
```
            <cache-container name="hibernate" default-cache="local-query" modules="org.hibernate.infinispan">
                <local-cache name="entity">
                    <transaction mode="NON_XA"/>
                    <heap-memory size="10000"/>
                    <expiration max-idle="100000"/>
                </local-cache>
                <local-cache name="local-query">
                    <transaction mode="NONE"/>
                    <heap-memory size="10000"/>
                    <expiration max-idle="100000"/>
                </local-cache>
                <local-cache name="timestamps">
                    <transaction mode="NONE"/>
                </local-cache>
            </cache-container>
```

--- a/clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/CacheConfigurationServiceConfigurator.java
+++ b/clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/CacheConfigurationServiceConfigurator.java
@@ -39,6 +39,7 @@ import org.infinispan.configuration.cache.PersistenceConfiguration;
 import org.infinispan.configuration.cache.StorageType;
 import org.infinispan.configuration.cache.TransactionConfiguration;
 import org.infinispan.distribution.group.Grouper;
+import org.infinispan.transaction.tm.EmbeddedTransactionManager;
 import org.jboss.as.clustering.controller.CapabilityServiceNameProvider;
 import org.jboss.as.clustering.controller.ResourceServiceConfigurator;
 import org.jboss.as.controller.OperationContext;
@@ -54,7 +55,6 @@ import org.wildfly.clustering.service.Dependency;
 import org.wildfly.clustering.service.ServiceConfigurator;
 import org.wildfly.clustering.service.ServiceSupplierDependency;
 import org.wildfly.clustering.service.SupplierDependency;
-import org.wildfly.transaction.client.ContextTransactionManager;
 
 /**
  * Builds a cache configuration from its components.
@@ -130,7 +130,7 @@ public class CacheConfigurationServiceConfigurator extends CapabilityServiceName
         }
         try {
             // Configure invocation batching based on transaction configuration
-            builder.invocationBatching().enable(tx.transactionMode().isTransactional() && (tx.transactionManagerLookup().getTransactionManager() != ContextTransactionManager.getInstance()));
+            builder.invocationBatching().enable(tx.transactionMode().isTransactional() && (tx.transactionManagerLookup().getTransactionManager() == EmbeddedTransactionManager.getInstance()));
         } catch (Exception e) {
             throw new CacheException(e);
         }

